Sources/RichTextKit/RichTextView_AppKit.swift

```diff
@@ -209,4 +211,21 @@ public extension RichTextView {
         textStorage
     }
 }
+
+// MARK: - Content Size
+
+extension NSTextView {
```

Comment:
Also I think this should not extend NSTextView rather  RichTextViewComponent itself. 

1. Abstraction for both platforms 
2. Better control over what is going on in custom RichTextViewComponent implementation.

Reply:
You're right. it should take both platforms into account. For my project it is necessary to have a way to know the height of the textview in order to place this richtexteditor view in a Scrollview or List (SwiftUI) between two other view elements. I think this would also be feasible for other users.